content/docs/command-reference/remote/modify.md

```diff
@@ -75,6 +75,13 @@ The following config options are available for all remote types:
   $ dvc remote modify localremote url /home/user/dvcstore
   ```
 
+- `jobs` - number of workers started. Accepts positive integers. The default
+  value is `4 * cpu_count()`. For SSH remotes, the default is `4`.
```

Comment:
"workers started" for what? What commands does it affect? Should we mention that it's the same as the `--jobs` option in some commands? Thanks!